docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf

```diff
@@ -158,6 +158,19 @@ create_table_as ::= 'CREATE' 'TABLE' [ 'IF' 'NOT' 'EXISTS' ] \
                            table_name [ '(' column_name { ',' column_name } ')' ] \
                            'AS' query [ 'WITH' [ 'NO' ] 'DATA' ] ;
 
+
+(* CREATE TRIGGER *)
+create_trigger = 'CREATE' 'TRIGGER' name
```

Comment:
```suggestion
create_trigger = 'CREATE' [ 'CONSTRAINT' ] 'TRIGGER' name
```

Reply:
Omitted intentionally as we don't support it yet.